internal/controller/apiextensions/composite/reconciler_test.go

```diff
@@ -517,7 +517,7 @@ func TestReconcile(t *testing.T) {
 						MockGet: test.NewMockGetFn(nil),
 						MockStatusUpdate: WantComposite(t, NewComposite(func(cr resource.Composite) {
 							cr.SetCompositionReference(&corev1.ObjectReference{})
-							cr.SetConditions(xpv1.ReconcileSuccess(), xpv1.Creating())
+							cr.SetConditions(xpv1.ReconcileSuccess(), xpv1.Creating().WithMessage("Unready resources: elephant"))
```

Comment:
we also need additional tests that:
* validate that names are really sorted (here we have just single resource) 
* we display a proper message when the number of unready resources are greater than 3

Reply:
done. Also compare https://github.com/crossplane/crossplane-runtime/pull/538 for a follow-up to replace this helper everywhere (we have like 5 or more of those now).